Modules/Battery/popup.swift

```diff
@@ -224,7 +224,7 @@ internal class Popup: NSView, Popup_p {
             self.dashboardBatteryView?.setValue(abs(value.level))
             
             self.levelField?.stringValue = "\(Int(abs(value.level) * 100)) %"
-            self.sourceField?.stringValue = value.powerSource
+            self.sourceField?.stringValue = "\(LocalizedString(value.powerSource))"
```

Comment:
I noticed it after merge. `LocalizedString` returns a String. So you don't need to pass it throw quotes. It could be simplified with this: `self.sourceField?.stringValue = LocalizedString(value.powerSource)`.

But it's ok)